app/pages/sections/order-items/catalogue-solutions/order-item/tiered/patient/ui-tests/general.ui.test.js

```diff
@@ -0,0 +1,361 @@
+
+  await t
+    .expect(solutionTableError.exists).notOk()
+    .click(saveButton);
+
+  await t
+    .expect(solutionTableError.exists).ok()
+    .expect(await extractInnerText(solutionTableError)).contains(content.errorMessages.QuantityRequired);
+});
+
+test('should anchor to the table when clicking on the error select practice size link in errorSummary', async (t) => {
+  await pageSetup({ ...defaultPageSetup, postRoute: true });
+  await t.navigateTo(pageUrl);
+
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .click(saveButton);
+
+  await t
+    .click(errorSummary.find('li a').nth(0))
+    .expect(getLocation()).eql(`${pageUrl}#quantity`);
+});
+
+test('should render solution table as errors with error message when no date is entered', async (t) => {
+  await pageSetup({ ...defaultPageSetup, postRoute: true });
+  await t.navigateTo(pageUrl);
+
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const solutionTableError = Selector('[data-test-id="solution-table-error"]').nth(1);
+  const dayInput = Selector('#deliveryDate-day');
+  const monthInput = Selector('#deliveryDate-month');
+  const yearInput = Selector('#deliveryDate-year');
+
+  await t
+    .expect(solutionTableError.exists).notOk()
+    .selectText(dayInput).pressKey('delete')
+    .expect(dayInput.hasClass('nhsuk-input--error')).notOk()
+    .selectText(monthInput).pressKey('delete')
+    .expect(monthInput.hasClass('nhsuk-input--error')).notOk()
+    .selectText(yearInput).pressKey('delete')
+    .expect(yearInput.hasClass('nhsuk-input--error')).notOk()
+    .click(saveButton);
+
+  await t
+    .expect(solutionTableError.exists).ok()
+    .expect(await extractInnerText(solutionTableError)).contains(content.errorMessages.DeliveryDateRequired);
+});
+
+test('should anchor to the table when clicking on the error select date link in errorSummary', async (t) => {
+  await pageSetup({ ...defaultPageSetup, postRoute: true });
+  await t.navigateTo(pageUrl);
+
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const dayInput = Selector('#deliveryDate-day');
+  const monthInput = Selector('#deliveryDate-month');
+  const yearInput = Selector('#deliveryDate-year');
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .selectText(dayInput).pressKey('delete')
+    .expect(dayInput.hasClass('nhsuk-input--error')).notOk()
+    .selectText(monthInput).pressKey('delete')
+    .expect(monthInput.hasClass('nhsuk-input--error')).notOk()
+    .selectText(yearInput).pressKey('delete')
+    .expect(yearInput.hasClass('nhsuk-input--error')).notOk()
+    .click(saveButton);
+
+  await t
+    .click(errorSummary.find('li a').nth(1))
+    .expect(getLocation()).eql(`${pageUrl}#deliveryDate`);
+});
```

Comment:
We don't need to support tiered prices yet so we can remove the code that relates to tiered pricing. However, keep a copy (either in a stash and/or a copy of the files) as we may be able to make use of it later.

Reply:
Removed tiered folder also kept a copy of the files.